src/core/order_manager.py

```diff
@@ -0,0 +1,25 @@
+# order manager mechanism
+
+from queue import Queue
+
+
+class OrderManager:
+    def __init__(self):
+        self._order_storage = {
```

Comment:
Could you rename to `order_status_to_order_queue_map`?

Reply:
done